### PR TITLE
Improvement to docs for send_file function

### PR DESCRIPTION
**Pull request by LABCAT**

### CODE REVIEW

File: src/Control/HTTPRequest.php [hunk 1]
```diff
@@ -468,6 +468,8 @@ public function offsetUnset($offset)
      * Construct an HTTPResponse that will deliver a file to the client.
      * Caution: Since it requires $fileData to be passed as binary data (no stream support),
      * it's only advisable to send small files through this method.
+     * this function needs to called inside the controllers response, eg
```
Comment (robbieaverill) [inline src/Control/HTTPRequest.php]:
```suggestion
     * This function needs to called inside the controller’s response, e.g.:
```

File: src/Control/HTTPRequest.php [hunk 2]
```diff
@@ -468,6 +468,8 @@ public function offsetUnset($offset)
...
+     * $this->setResponse(HTTPRequest::send_file('the content', 'filename.txt'));
```
Comment (robbieaverill) [inline src/Control/HTTPRequest.php]:
```suggestion
     * <code>$this->setResponse(HTTPRequest::send_file('the content', 'filename.txt'));</code>
```

File: src/Control/HTTPRequest.php (hunk 1 above)
Comment (ScopeyNZ) [reply src/Control/HTTPRequest.php]: This suggestion was resolved but not applied?

Comment (LABCAT) [reply src/Control/HTTPRequest.php]: I think it is resolved now.

File: src/Control/HTTPRequest.php [hunk 3]
```diff
@@ -468,7 +468,7 @@ public function offsetUnset($offset)
      * Construct an HTTPResponse that will deliver a file to the client.
      * Caution: Since it requires $fileData to be passed as binary data (no stream support),
      * it's only advisable to send small files through this method.
-     * this function needs to called inside the controllers response, eg
+     * This function needs to called inside the controller’s response, e.g.:
```
Comment (michalkleiner) [inline src/Control/HTTPRequest.php]: .... to be called ...

Comment (ScopeyNZ) [reply src/Control/HTTPRequest.php]:
```suggestion
     * This function needs to be called inside the controller’s response, e.g.:
```

**Pull request merged**